src/components/DashBody.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react'
-import DashWidget from './DashWidget'
+import TextWidget from './TextWidget'
 import '../styles/dash_body.css'
 
 class DashBody extends Component {
```

Comment:
Can we rename this to `DashboardBody`? I prefer the explicit name rather than attempting to minify.